packages/date-picker/theme/material/vaadin-date-picker-overlay-content-styles.js

```diff
@@ -54,11 +49,6 @@ registerStyles(
       width: 24px;
       height: 24px;
       text-align: center;
-    }
-
-    [part='clear-button'],
```

Comment:
Here I merged two selectors because `padding` and `color` for `year-toggle-button` are specified separately.